docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,703 @@
+
+//Additional line break to resolve mod docs generation error
+
+[id="proc-install-rhoas-ansible-collection_{context}"]
+=== Installing the RHOAS Ansible collection
+
+[role="_abstract"]
+You can use the https://galaxy.ansible.com/rhoas/rhoas[RHOAS Ansible collection] to fully manage your Kafka environment with Ansible. You can create, retrieve info, configure, deploy, and delete the following {product-long-kafka} resources:
+
+* Kafka instances
+* Service accounts
+* Kafka topics
+* Access Control Lists (ACLs)
+
+.Prerequisites
+
+* Python 3.9 or later is installed.
+* https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html?extIdCarryOver=true&sc_cid=701f2000001Css5AAC[Ansible^] 2.9 or later is installed.
+* This collection requires some additional dependencies to be installed. To do this, enter the following command.
++
+[source,shell]
+----
+$ pip install rhoas-sdks
+pip install python-dotenv
+----
+NOTE: Use `pip install rhoas-sdks --force reinstall`  to ensure that the most up to date versions of the SDKs are installed.
++
+
+* (Optional) This collection works best when used in a Python virtual environment. To create and activate a Python virtual environment, enter the following command:
++
+[source,shell]
+----
+$ python3 -m venv rhoas
+. rhoas/bin/activate
+----
++
+
+.Procedure
+
+. Open a terminal and enter the following command:
++
+[source,shell]
+----
+$ ansible-galaxy collection install rhoas.rhoas
+----
++
+By default, ansible-galaxy collection install uses https://galaxy.ansible.com as the Galaxy server (as listed in the ansible.cfg file under GALAXY_SERVER). You do not need any further configuration. The install command automatically appends the path `ansible_collections` to the one specified with the `-p` option unless the parent directory is already in a folder called `ansible_collections`.
+. Once installed, you can reference a collection content by its fully qualified collection name (FQCN).
+
+NOTE: The `create_kafka` command in the task needs to be the FQCN of the module for collection development, for example
+`rhoas.rhoas.create_kafka`.
+
+
+[id="proc-using-rhoas-ansible-collection_{context}"]
+=== Using the RHOAS Ansible collection
+
+.Prerequisites
+* You have a Red Hat account.
+* You have an offline token that is accessible to Ansible.
+
+[NOTE]
+The offline token is a refresh token with no expiry and can be used by non-interactive processes to provide an access token for {product-long-rhoas} to authenticate the user. The token is an OpenShift offline token and can be found at https://cloud.redhat.com/openshift/token.
+
+* Two further environment variables are used for the collection to work. These environment variables can be placed in a `.env` file. If neither of these environment variables are set, the collection will default to the URLs shown in the examples.
```

Comment:
This is more of an overall comment, but here is a good example. We should try to eliminate as much passive voice as possible, particularly given that this guide is about a range of actions that become automated based on a playbook. It will be quite important throughout this guide to separate any user actions from those performed by the playbook.

Therefore:

"The collection requires the following two variables to work:

- API_BASE_HOST - The base host for the API. This is the base URL for the API. For example, https://api.openshift.com.
- SSO_BASE_HOST - The base host for the SSO. This is the base URL for the SSO. For example, https://sso.redhat.com/auth/realms/redhat-external.

You can define the environment variables shown in an environment variable (`.env`) file. If you don't explicitly define these environment variables, the collection uses the URLs shown in the examples."